src/main/java/model/LottoCompareResult.java

```diff
@@ -0,0 +1,45 @@
+package model;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class LottoCompareResult {
+
+    private static final int NO_CONDITION_RANK_INDEX = 0;
+    private static final int NO_CONDITION_RANK_SIZE = 1;
+
+    private final long equalCount;
+    private final boolean isBonusNumberSame;
+
+    public LottoCompareResult(final long count, final boolean isSame) {
+        this.equalCount = count;
+        this.isBonusNumberSame = isSame;
+    }
+
+    public Optional<LottoRank> getResultRank() {
+        List<LottoRank> sameCountRank = Arrays.stream(LottoRank.values())
+            .filter(lottoRank -> lottoRank.hasSameCount(equalCount))
+            .toList();
```

Comment:
컨벤션 기준 가독성을 위한 개행 상황에서의 들여쓰기는 탭 2번, 또는 스페이스바 8번 입니다!
```suggestion
        List<LottoRank> sameCountRank = Arrays.stream(LottoRank.values())
                .filter(lottoRank -> lottoRank.hasSameCount(equalCount))
                .toList();
```

Reply:
헉 이부분을 놓쳤었군요 감사합니다!